src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
-body {
+@import "./utils/variables";
+@import "./utils/mixins";
+
+.page {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) 3rd point

Reply:
I thought this task is a separate block already
done